server/channels/store/store.go

```diff
@@ -724,13 +725,12 @@ type JobStore interface {
 	UpdateOptimistically(job *model.Job, currentStatus string) (bool, error)
 	UpdateStatus(id string, status string) (*model.Job, error)
 	UpdateStatusOptimistically(id string, currentStatus string, newStatus string) (bool, error)
-	Get(id string) (*model.Job, error)
-	GetAllPage(offset int, limit int) ([]*model.Job, error)
-	GetAllByType(jobType string) ([]*model.Job, error)
-	GetAllByTypeAndStatus(jobType string, status string) ([]*model.Job, error)
-	GetAllByTypePage(jobType string, offset int, limit int) ([]*model.Job, error)
-	GetAllByTypesPage(jobTypes []string, offset int, limit int) ([]*model.Job, error)
-	GetAllByStatus(status string) ([]*model.Job, error)
+	Get(c *request.Context, id string) (*model.Job, error)
+	GetAllByType(c *request.Context, jobType string) ([]*model.Job, error)
+	GetAllByTypeAndStatus(c *request.Context, jobType string, status string) ([]*model.Job, error)
+	GetAllByTypePage(c *request.Context, jobType string, offset int, limit int) ([]*model.Job, error)
+	GetAllByTypesPage(c *request.Context, jobTypes []string, offset int, limit int) ([]*model.Job, error)
+	GetAllByStatus(c *request.Context, status string) ([]*model.Job, error)
```

Comment:
The first `*request.Context` in the store!

Reply:
Added my comment above :)